src/axom/primal/geometry/Triangle.hpp

```diff
@@ -330,18 +331,29 @@ class Triangle
   }
 
   /*!
-   * \brief Returns whether Point P is in the triangle for some 3d Triangle
-   * \return true iff P is in the triangle
-   * \see primal::Point
+   * \brief Returns whether Point \a p is contained within the triangle
+   * 
+   * \return true if P is in the triangle
```

Comment:
Should this instance of "P" be replaced with "\a p" to show it is an argument? Or "point p" to keep capitalization consistent? Same thing in the doxygen for the contains method.

Reply:
Since the parameter is `p`, I went for the lower case option. Not sure why the original had a capital P.